samples/snippets/get_common_metadata.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Analytics Data API sample application retrieving dimension and metrics
+metadata.
+
+See https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/getMetadata
+for more information.
+"""
+# [START analyticsdata_get_common_metadata]
+from google.analytics.data_v1beta import BetaAnalyticsDataClient
+from google.analytics.data_v1beta.types import GetMetadataRequest
+from google.analytics.data_v1beta.types import MetricType
+
+
+def run_sample():
+    """Runs the sample."""
+    get_common_metadata()
+
+
+def get_common_metadata():
+    """Retrieves dimensions and metrics available for all Google Analytics 4
+    properties."""
+    client = BetaAnalyticsDataClient()
+
+    # Set the Property ID to 0 for dimensions and metrics common
+    # to all properties. In this special mode, this method will
+    # not return custom dimensions and metrics.
+    property_id = 0
+    request = GetMetadataRequest(name=f"properties/{property_id}/metadata")
+    response = client.get_metadata(request)
+
+    print("Dimensions and metrics available for all Google Analytics 4 properties:")
+    print_get_metadata_response(response)
+
+
+def print_get_metadata_response(response):
+    """Prints results of the getMetadata call."""
+    # [START analyticsdata_print_get_metadata_response]
```

Comment:
nit: Do you still want this region tag here if the wider one includes this section?